src/main/java/org/jclouds/vcloud/director/v1_5/compute/functions/ImageForVAppTemplate.java

```diff
@@ -97,10 +97,10 @@ public Image apply(QueryResultVAppTemplateRecord from) {
           try {
               builder.location(findLocationForResource.apply(reference));
           } catch (NoSuchElementException e) {
-              logger.warn("The image doesn't have a location metadata attached and cannot be prioritized", e);
+              logger.debug("The image {} doesn't have location metadata attached", from, e);
```

Comment:
jclouds logging requires `%s` rather than the slf4j style of `{}`.

Also, the `e` at the end is presumably ignored? There is no debug or info method that takes an exception. But you can do a toString on the exception by including a second `%s`.